.internal-ci/util/wait-for-full-service.sh

```diff
@@ -4,22 +4,36 @@ set -e
 set -o pipefail
 
 
-echo "Checking block height - wait for full-service to start"
-sleep 15
+echo "-- Checking block height - wait for full-service to start"
+
+TARGET_FS_URL=${TARGET_FS_URL:-"http://localhost:9090/wallet/v2"}
 
 curl_post()
 {
-    curl --connect-timeout 2 -sSL -X POST -H 'Content-type: application/json' http://localhost:9090/wallet/v2 --data '{"method": "get_wallet_status", "jsonrpc": "2.0", "id": 1}' 2>/dev/null
+    curl --connect-timeout 2 -sSfL -X POST -H 'Content-type: application/json' "${TARGET_FS_URL}" --data '{"method": "get_wallet_status", "jsonrpc": "2.0", "id": 1}' 2>/dev/null
 }
 
+count=0
+while ! curl_post
+do
+    echo "-- Waiting for full-service to respond"
+    count=$((count + 1))
+    if [[ "${count}" -gt 300 ]]
+    then
+        echo "Full-service did not start"
+        exit 1
+    fi
+    sleep 2
+done
+
 # wait for blocks
 wallet_json=$(curl_post)
 network_block_height=$(echo "${wallet_json}" | jq -r .result.wallet_status.network_block_height)
```

Comment:
nb: if using `get_network_status` then this would be r`esult.network_status.network_block_height` and `result.network_status.local_block_height `instead.